include/triton/core/tritonserver.h

```diff
@@ -1761,6 +1761,17 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error*
 TRITONSERVER_ServerOptionsSetModelLoadThreadCount(
     TRITONSERVER_ServerOptions* options, unsigned int thread_count);
 
+/// Provide a log output file.
+///
+/// \param options The server options object.
+/// \param file a string defining the file where the log outputs will be saved.
+/// An "empty" string for the file name will cause triton to direct logging
```

Comment:
don't double quote empty, also check `nullptr` and log to console if `nullptr` is provided.